tests/unit/fish_completion_test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+. ./tests/unit/utils.sh --source-only
+
+function suite()
+{
+  suite_addTest "testKwCompletion"
```

Comment:
Maybe use a more meaningful name for this test, for example: `testKwCompletionForFish`.

P.s.: I never used this suite_addTest, do we really need it here?